core-bundle/contao/themes/flexible/styles/main.css

```diff
@@ -646,6 +646,7 @@ h2.sub_headline {
 
 .tl_filter > * {
 	margin: 0;
+	width: 100%;
```

Comment:
What does this do?

Reply:
It makes sure the elements are displayed full width on smaller viewports. It's needed now that the child element (the host `div`) has not explicitly set `width: 100%` anymore.